charts/wikijs/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+appVersion: 0.0.1
```

Comment:
It's okay to start the version at 1.0.0

Reply:
I have updated it to 1.0.0. Can you also look at https://github.com/k8s-at-home/library-charts/pull/5 ? I cannot use this persistence with this chart because of the bug in the persistence template.